.coveragerc

```diff
@@ -7,7 +7,6 @@ source = homeassistant
 omit =
     homeassistant/__main__.py
     homeassistant/helpers/signal.py
-    homeassistant/helpers/backports/*
```

Comment:
This change should not be in this PR. It was included in https://github.com/home-assistant/core/pull/115044
Unfortunately, I can't rebase your PR easily right now. So, it would be great if you can do that / remove this change here.